packages/eslint-plugin/src/rules/member-naming.ts

```diff
@@ -94,7 +94,7 @@ export default util.createRule<Options, MessageIds>({
     ): void {
       if (
         node.type === AST_NODE_TYPES.MethodDefinition &&
-        node.kind === 'constructor'
+        node.kind === AST_NODE_TYPES.constructor
```

Comment:
this is not correct

Reply:
Yeah, I've got to push my latest re-lint.

It's because I forgot to run `yarn build` :)